src/components/form/Select.js

```diff
@@ -3,44 +3,33 @@ import PropTypes from 'prop-types'
 import { Field } from 'react-final-form'
 import FormControl from '@material-ui/core/FormControl'
 import InputLabel from '@material-ui/core/InputLabel'
-import FormHelperText from '@material-ui/core/FormHelperText'
 import { default as MUISelect } from '@material-ui/core/Select'
 import { withStyles } from '@material-ui/core/styles'
+import { formInput, formOptions } from '../../utils/react/propTypes'
 import { styles } from './styles'
+import { ErrorText } from './buildingBlocks/ErrorText'
 
 export const Select = withStyles(styles)(props => (
     <div className={props.classes.container}>
         <Field name={props.name} placeholder={props.placeholder}>
             {({ input, meta, placeholder }) => (
```

Comment:
Would this perhaps be a sensible place to use the spread operator, i.e.
```javascript
fieldProps => <StyledSelect {...props} {...fieldProps} />
```

Reply:
Hmm there are only 3 props of the `Select` that are passed to the Style select..
I could do that but think it causes more confusion in this case

Reply:
Agreed 👍